pyanaconda/modules/storage/bootloader/installation.py

```diff
@@ -141,6 +142,34 @@ def run(self):
             raise BootloaderInstallationError(str(e)) from None
 
 
+class CreateBLSEntriesTask(Task):
+    """The installation task that creates BLS entries."""
+
+    def __init__(self, storage, payload_type, kernel_versions, sysroot):
+        """Create a new task."""
+        super().__init__()
+        self._storage = storage
+        self._payload_type = payload_type
+        self._versions = kernel_versions
+        self._sysroot = sysroot
+
+    @property
+    def name(self):
+        return "Create the BLS entries"
+
+    def run(self):
+        """Run the task."""
+        if self._payload_type not in PAYLOAD_LIVE_TYPES:
+            log.debug("Only live payloads require this fix.")
```

Comment:
Is this correct ? Either it is a copy-paste error or the error message should be changed to something like "BLS does/does not work on X".

Reply:
It is a payload-specific fix. It doesn't mean that BLS doesn't work.